docs/en/spark/configuration/sink-plugins/Iceberg.md

```diff
@@ -0,0 +1,62 @@
+# Iceberg
+
+> Sink plugin: Iceberg [Spark]
+
+## Description
+
+Write data to Iceberg.
+
+## Options
+
+| name           | type   | required | default value |
+| -------------- | ------ | -------- | ------------- |
+| [path](#path)  | string | yes      | -             |
+| [saveMode](#saveMode) | string | yes | -             |
+| [target-file-size-bytes](#target-file-size-bytes) | long | no      | -   |
+| [check-nullability](#check-nullability) | bool | no| - |
+| [snapshot-property.custom-key](#snapshot-property.custom-key) | string | no| - |
+| [fanout-enabled](#fanout-enabled) | bool | no| - |
+| [check-ordering](#check-ordering) | bool | no| - |
+
+
+Refer to [iceberg write options](https://iceberg.apache.org/docs/latest/spark-configuration/) for more configurations.
+
+### path
+
+Iceberg table location.
+
+### saveMode
+
```

Comment:
@ououtt `saveMode` I guess it's better to have a default value

Reply:
Is `saveMode` only these two? I think you can write a link that looks for `saveMode` after

Reply:
The `Options` table above is best arranged neatly

Reply:
iceberg only support append and overwrite. and I have set the default value of saveMode to append and format the options table.